### DISCUSSION TITLE
Export Data

### DISCUSSION THREAD

**Discussion by SwaiEnt** [Marekkon5/onetagger#151] [Q&A]
Is there any way to export all track information (Auto Tag) from a folder or individual track to an Excel or Sheets (csv.)?

**Comment by Marekkon5** [comment]
You mean the original data or the fetched data?

**Reply by SwaiEnt** [reply]
Original and updated metadata.

**Reply by Marekkon5** [reply]
There is no way to do this currently, and I have no idea how this should be used / implemented / what's the purprose and I do think this is out of scope for 1T..
